Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Operating Systems
+permalink: /OS/
+nav_order: 80
+parent: Installing and configuring
+has_children: yes
+has_toc: yes
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Heads and Operating Systems
+====
+
+Since a heads only system would be very limited you will need to install at least one Operating System.  The installation procedures vary greatly depending on which distro you choose, which version of that distro, and if you are encrypting partitions.  Some choices you will have to make include:
+
+* partition table
+* LUKS encryption (yes/no)
+* LUKS version (1 or 2)
+* single vs multiboot
+* simple distro vs hypervisor
+
+All together there are a lot of technical decisions.  This document will attempt to inform you enough to make the decisions.
+
+
+partition table
+----
+
+heads currently requires a separate boot partition which is a common configuration for Linux.  When you partition storage make sure to create this or have the OS installer do it for you.  Generally, the boot partition is much smaller than the other OS partition(s).  You may keep everything else on 1 partition to make it easier to manage but the kernel and initrd on the boot partition can mount any number of others--encrypted or not.
+
+The other choices mentioned below may also affect the partition layout you choose.
+
+
+LUKS encryption
+----
+
+If you have taken the trouble to install and use heads it is likely that you also want to use encryption to protect your data on storage devices.  LUKS is the most commonly used format in a heads based system.  
+
+### encrypted boot
+
+if heads can get to your decryption key (from the TPM or stored internally) you may encrypt the boot partition.  This is non trivial and may be more trouble than the alternatives.  See 'encrypted root'.
+
+### encrypted root
+
+Since the root partition is the end result of the boot process you may see how encrypting it makes the boot process complicated.  There are multiple schemes for making this setup work and more will probably be invented.  The key issue is deciding where to store the decryption key so that the root partition can be accessed during boot.  Some options are:
+
+* key in BIOS
+* key in TPM
+* key in /boot
+* key on removable media
+* key encrypted by smart card
+
+These can also be combined for various mitigations against different attack vectors.  For example, the [Pureboot](https://docs.puri.sm/PureBoot.html) model (based on heads) uses a smart card to encrypt the LUKS decryption key and stores it on /boot which is not encrypted.  In addition, the smartcard is used to sign the important files in /boot which are verified at each boot.  
```

Comment:
Point  to doc. dont duplicate with interpretation. 
https://docs.puri.sm/PureBoot/LibremKeyLUKS.html

Reply:
WIP That link has none of the information from lines 55 to 63.  Is the information wrong?  

This is a discussion of handling encrypted root partitions.  heads may not be involved at all but it could be if using the TPM approach.  This warrants some documentation on the options available and I included links to Pureboot as an example of 1 approach.